model/pubspec.yaml

```diff
@@ -56,6 +56,11 @@ dependencies:
 
   dartz: 0.10.1
 
+  duration: 4.0.3
+
+  email_recovery:
+    path: ../email_recovery
```

Comment:
IMO, `model` no need to be aware about `email_recovery`. Why not move related stuffs back to `email_recovery`

Reply:
as you want ! @dab246 is it ok with you ?

Reply:
I agree with @hoangdat opinion

Reply:
> as you want ! @dab246 is it ok with you ?

You can create a `session_extension` in `email_recovery` without having to use `session_extension` in the model.

Reply:
you're right ! i did that